src/lib/pages/frontsite/Documents/DocumentDetails/DocumentDetails.js

```diff
@@ -180,6 +181,7 @@ class DocumentDetails extends Component {
     if (hasError && _get(error, 'response.status') === 404) {
       return <NotFound />;
     }
+    const autofocus = screenIsWiderThan(Breakpoints.computer);
```

Comment:
you could add this inside of SearchBarILS, for the component to recognise when it is suppose to use autofocus, not in every parent component that uses it. It would reduce the amount of code.

Reply:
Yes, but since we only want this on the frontsite I thought it would be "cleaner" to not force this behaviour in the backoffice. Anyway if backoffice is not supossed to be responsive I guess we can move it inside SearchBarILS.

Reply:
the backoffice is not supposed to be used with responsive. you can still add it as an optional prop to SearchBar and SearchBarILS to have the control to turn off/on the focus, but  IMO the check if the screen is wide or not should be only done in the SeachBar, not the parent components (because it's not their purpose to know about details of displaying the SearchBar)

Reply:
Yes, you are absolutely right about your point, I will change it. Thanks!